sql/item_latents.sql

```diff
@@ -2715,4 +2715,46 @@ INSERT INTO `item_latents` VALUES(14122, 68, 8, 52, 3); -- Kyahan: Evasion +8 in
 INSERT INTO `item_latents` VALUES(14005, 25, 8, 52, 3); -- Tekko: Accuracy +8 in Water weather
 INSERT INTO `item_latents` VALUES(14026, 25, 9, 52, 3); -- Hailstorm Tekko: Accuracy +9 in Water weather
 INSERT INTO `item_latents` VALUES(14027, 25, 10, 52, 3); -- Hailstorm Tekko +1: Accuracy +10 in Water weather
-INSERT INTO `item_latents` VALUES(16883, 25, 10, 52, 3); -- Spear: Accuracy +10 in Water weather
\ No newline at end of file
+INSERT INTO `item_latents` VALUES(16883, 25, 10, 52, 3); -- Spear: Accuracy +10 in Water weather
+
+-- -------------------------------------------------------
+-- Rune equips
+-- -------------------------------------------------------
+INSERT INTO `item_latents` VALUES(18206, 25, 5, 56, 0); -- Rune Chopper +5 Acc.
+INSERT INTO `item_latents` VALUES(18206, 167, 9, 56, 0); -- Rune Chopper +9% haste
+INSERT INTO `item_latents` VALUES(18206, 369, -3, 56, 0); -- Rune Chopper -3MP/tic
+INSERT INTO `item_latents` VALUES(17333, 10, 1, 56, 0); -- Rune Arrow +1 VIT
+INSERT INTO `item_latents` VALUES(17333, 24, 10, 56, 0); -- Rune Arrow +10 R.Atk.
+INSERT INTO `item_latents` VALUES(17333, 369, -1, 56, 0); -- Rune Arrow -1MP/tic
+INSERT INTO `item_latents` VALUES(16647, 23, 5, 56, 0); -- Rune Axe +5 Atk.
+INSERT INTO `item_latents` VALUES(16647, 370, 5, 56, 0); -- Rune Axe +5HP/tic
+INSERT INTO `item_latents` VALUES(16647, 369, -3, 56, 0); -- Rune Axe -3MP/tic
+INSERT INTO `item_latents` VALUES(16408, 8, 7, 56, 0); -- Rune Baghnakhs +7 STR
+INSERT INTO `item_latents` VALUES(16408, 291, 1, 56, 0); -- Rune Baghnakhs +1 Counter
+INSERT INTO `item_latents` VALUES(16408, 369, -4, 56, 0); -- Rune Baghnakhs -4MP/tic
+INSERT INTO `item_latents` VALUES(12742, 68, 5, 56, 0); -- Rune Bangles +5 Eva.
+INSERT INTO `item_latents` VALUES(12742, 1, 32, 56, 0); -- Rune Bangles +32 Def.
+INSERT INTO `item_latents` VALUES(12742, 369, -4, 56, 0); -- Rune Bangles -4MP/tic
+INSERT INTO `item_latents` VALUES(16563, 9, 5, 56, 0); -- Rune Blade +5 DEX
+INSERT INTO `item_latents` VALUES(16563, 366, 4, 56, 0); -- Rune Blade DMG: 43
+INSERT INTO `item_latents` VALUES(16563, 369, -4, 56, 0); -- Rune Blade -4MP/tic
+INSERT INTO `item_latents` VALUES(17158, 26, 5, 56, 0); -- Rune Bow +5 R. Acc.
+INSERT INTO `item_latents` VALUES(17158, 27, -2, 56, 0); -- Rune Bow -2 Enmity
+INSERT INTO `item_latents` VALUES(17158, 369, -4, 56, 0); -- Rune Bow -4MP/tic
+INSERT INTO `item_latents` VALUES(18084, 9, 6, 56, 0); -- Rune Halberd +6 DEX
+INSERT INTO `item_latents` VALUES(18084, 288, 5, 56, 0); -- Rune Halberd +5% Dbl.Atk.
+INSERT INTO `item_latents` VALUES(18084, 369, -4, 56, 0); -- Rune Halberd -4MP/tic
+INSERT INTO `item_latents` VALUES(17461, 23, 10, 56, 0); -- Rune Rod +10 Atk.
+INSERT INTO `item_latents` VALUES(17461, 112, 6, 56, 0); -- Rune Rod +6 Healing Magic Skill
+INSERT INTO `item_latents` VALUES(17461, -4, 56, 0); -- Rune Rod -4MP/tic
```

Comment:
The correction here is present in my file, i'm currently looking into why this is still off.